app/views/manage_assessments/outcome_coverages/_outcome_coverage.html.erb

```diff
@@ -4,3 +4,14 @@
 <p class="class-card-outcome class-card-outcome-nickname">
   <%= outcome_coverage.outcome_nickname %>
 </p>
+<% if outcome_coverage.assignment.present? %>
+  <p>
+    <%= t(".assignment_name", name: outcome_coverage.assignment.name) %>
+    <%= t(".assignment_problem",
+        problem: outcome_coverage.assignment.problem) if outcome_coverage.assignment.problem.present? %>
```

Comment:
Now that problem is optional we only want to display the text up to the word `on` when a problem is not given. 
![screen shot 2017-05-23 at 1 48 56 pm](https://cloud.githubusercontent.com/assets/9501674/26368111/a05e1078-3fbe-11e7-98b2-3518f70dd9fb.png)